packages/presentation/lib/screen/cast_list/cast_crew_bloc.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:presentation/base/bloc.dart';
+import 'package:presentation/screen/cast_list/cast_crew_data.dart';
+import 'package:presentation/screen/cast_list/cast_crew_screen.dart';
+
+abstract class CastCrewBloc
+    extends Bloc<CastCrewScreenArguments, CastCrewData> {
+  factory CastCrewBloc() => CastCrewBlocImpl();
+}
+
+class CastCrewBlocImpl extends BlocImpl<CastCrewScreenArguments, CastCrewData>
+    implements CastCrewBloc {
+  CastCrewBlocImpl();
+
+  CastCrewData _stateData = CastCrewData.init();
+
+  void _updateData({
+    bool? isLoading,
+    CastCrewData? data,
+  }) {
+    handleData(
+      data: _stateData,
+      isLoading: isLoading,
+    );
+  }
+
+  @override
+  void initArgs(CastCrewScreenArguments arguments) {
+    _updateData(isLoading: true);
```

Comment:
there no api calls, do you needed this? there no api calls, do you needed this? data will appear on the ui immediately.

Reply:
it didn't work without it

Reply:
need to investigate why. it looks redundant